### ISSUE TITLE
Add release notes from merge tool

### ISSUE THREAD

**Issue by guibranco** [guibranco/BancosBrasileiros#319]
**Is your feature request related to a problem? Please describe.**
In the release notes add the change log relative to the current changes.

**Describe the solution you'd like**
Read the content of the file `release-notes.txt` that will be generated by [Add change log to release notes](https://github.com/guibranco/BancosBrasileiros-MergeTool/issues/26)

**Comment by shivathapaa** [comment]
Hi @guibranco, I would love to work on this issue #319.

Actually, I checked the repo and found a better long-term solution. If I would suggest, then I would like to add a release-changelog.md file which will follow the template you set for, and every time a release is made the action will run automatically and update the updated changes from the release-changelog.md file. The contents to be updated in the release description will be separated by the delimiter every time a release is made.

So, basically the release-changelog.md will keep the history of the changelog and also will support for updating the release description.

I hope you understand the idea I am describing. If you would assign this issue I will push the working solution for the enhancement you asked for.
Thanks.

**Comment by guibranco** [comment]
I don't understand well, you want to make `relase-changelog.md` log lived ? But we already have the `changelog.md` with this. Would you replace it (and why) or I missed something here ?

**Comment by shivathapaa** [comment]
Sorry for the misunderstanding, I think I may have misunderstood the issue. If you don't mind can you please elaborate on the issue so that I can try to contribute to the project. Thanks.

**Comment by guibranco** [comment]
Sure, a file named `release-notes.txt/release-note.md` (the name don't matter right now) will be generated by the [Merge Tool](https://github.com/guibranco/BancosBrasileiros-MergeTool/issues/26) every time the workflow [Daily Updates](https://github.com/guibranco/BancosBrasileiros/actions/workflows/daily-updates.yml) runs (if data is updated only). 

So what I need is to create a release (that is currently created by AppVeyor) and update it`s description with the content of this file (that was not committed). So basically, that`s the problem. 

I am thinking about changing the pipeline from AppVeyor entirely to GitHub Actions, or maybe adding this release-notes content as a artifact of the workflow run and access it after the release creation (what is a complex operation at all)

**Comment by shivathapaa** [comment]
Hi @guibranco ,

Thank you so much for your prompt response to this issue. I truly appreciate your help.

However, after carefully reviewing the issue and the provided description, I've come to realize that it's more complex than I initially thought, and I'm unable to dedicate the time required to address them effectively.
I'm genuinely sorry that I won't be able to follow through and complete the fix. I had hoped to make a meaningful contribution to this project.
I'll be looking forward to assisting with other issues.

Thank you once again for your understanding. I greatly appreciate your dedication to maintaining this project.

**Comment by guibranco** [comment]
Surr, no problem, I understand.


I would also like to apologize for not being clear and giving the appropriate context when opening the issue.
